docs/index.md

```diff
@@ -34,19 +35,22 @@ and [How to use the playground](./how-to-use-the-playground.md).
 
 ## Getting started
 
-To get started with Gravitino, see [Getting started](./getting-started.md) for the details.
+To get started with Gravitino, see [Getting started](./getting-started) for the details.
 
-* [Getting started locally](./getting-started.md#getting-started-locally): a quick guide to starting
+* [Getting started locally](./getting-started/#local): a quick guide to starting
```

Comment:
Can you please keep the `.md` suffix, so the link can be worked in Github preview.

Reply:
Okay, will check it.